tests/integration_tests/security/test_seccomp.py

```diff
@@ -229,3 +230,30 @@ def test_no_seccomp(test_microvm_with_api):
     test_microvm.start()
 
     utils.assert_seccomp_level(test_microvm.jailer_clone_pid, "0")
+
+
+# The possible Firecracker --seccomp-level values.
+SECCOMP_LEVELS = ["0", "1", "2"]
+
+# Map FC seccomp-level to kernel seccomp-level.
+# Note that level 1 also maps to kernel level 2, which stands for
+# any custom BPF filter.
+KERNEL_LEVEL = {"0": "0", "1": "2", "2": "2"}
+
+
+@pytest.mark.parametrize(
+    "level",
+    SECCOMP_LEVELS
+)
+def test_seccomp_level(test_microvm_with_api, level):
+    """Test Firecracker --seccomp-level 0."""
```

Comment:
The test description is not consistent to the behavior. 
`"""Test Firecracker --seccomp-level level."""`